### PR TITLE
backend/coins/btc/headers: add Close()

### PR DESCRIPTION
**Pull request by benma**
Shutdown goroutine and close db.

### CODE REVIEW

File: backend/coins/btc/blockchain/mocks/mock.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
+package mocks
```
Comment (x1ddos) [inline backend/coins/btc/blockchain/mocks/mock.go]: License headers and pkg doc comment?

File: backend/coins/btc/blockchain/mocks/mock.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+import (
+	chainhash "github.com/btcsuite/btcd/chaincfg/chainhash"
+	wire "github.com/btcsuite/btcd/wire"
+	btcutil "github.com/btcsuite/btcutil"
+	blockchain "github.com/digitalbitbox/bitbox-wallet-app/backend/coins/btc/blockchain"
+)
+
+// BlockchainMock implements Interface for use in tests.
```
Comment (x1ddos) [inline backend/coins/btc/blockchain/mocks/mock.go]: I'd say "implements blockchain.Interface". It was clear which interface as long as this BlockchainMock struct was in the same blockchain package, but it's split into "mocks" now.

File: backend/coins/btc/blockchain/mocks/mock.go (hunk 1 above)
Comment (benma) [reply backend/coins/btc/blockchain/mocks/mock.go]: Done

File: backend/coins/btc/blockchain/mocks/mock.go (hunk 2 above)
Comment (benma) [reply backend/coins/btc/blockchain/mocks/mock.go]: correct, done

**Pull request merged**